Sources/Testing/Test+Macro.swift

```diff
@@ -149,7 +149,7 @@ extension Test {
   ///
   /// - Warning: This type alias is used to implement the `@Test` macro. Do not
   ///   use it directly.
-  public typealias __ParameterInfo = (firstName: String, secondName: String?)
+  public typealias __Parameter = (firstName: String, secondName: String?)
```

Comment:
We determined this shouldn't be the same type, right?

Reply:
Right, it makes the macro expansion simpler, avoids needing to expose `init`, and also we derive the `index` value from the position in this array